### DISCUSSION TITLE
참가 신청으로 인한 팀페이지 리셋

### DISCUSSION THREAD

**Discussion by kyalivia** [devrel-kr/HackaLearn#186] [1 Q&A - HackaLearn 운영] [answered]
참가 신청으로 인해 자동으로 팀페이지가 생성이 되면서, 이미 생성된 팀페이지에 작성된 정보(챌린지)가 지워졌습니다.
이미 PR을 날린 팀원들은 다시 PR을 날려야 하나요?

**Comment by justinyoo** [comment, marked as answer]
@Kyalivia 추가 팀원 영입으로 인해 팀 페이지가 리셋됐다면 다시 해당 부분들을 올려주세요. 그리고, PR 생성시 이 부분을 언급해 주시면 확인후 조치하겠습니다.

**Reply by kyalivia** [reply]
네, 알겠습니다.
